JDI.Light/JDI.Light.Tests/Tests/Composite/TableTests.cs

```diff
@@ -96,5 +96,26 @@ public void HugeTableSearchByColumnNumbersHasValuesTest()
             Assert.AreEqual("Brian Meyer;(016977) 0358;mollis.nec@seddictumeleifend.co.uk;Houston",
                 row.GetValue());
         }
+
+        [Test]
+        public void TableChainTest()
+        {
+            PerformancePage.Open();
+            PerformancePage.CheckOpened();
+            PerformancePage.UsersTable.AssertThat()
+                .Size(400)
+                .Size(Is.GreaterThan(399))
+                .Size(Is.LessThanOrEqualTo(400))
+                .Size(Is.GreaterThanOrEqualTo(400))
+                .Size(Is.LessThan(401))
+                .HasRowWithValues(
+                    HasValue("Brian Meyer", InColumn("Name")),
+                    HasValue("mollis.nec@seddictumeleifend.co.uk", InColumn("Email")))
+                .NotEmpty()
+                .RowsWithValues(3, ContainsValue("Baker", InColumn(1)))
+                .HasColumn("Email")
+                .HasColumns(new[] {"Name", "City"})
+                .Columns(Is.SubsequenceOf(new[] {"Name", "City", "Phone", "Email", "Address"}));
+        }
     }
 }
```

Comment:
Please, remove a new line at the end of file. It doesn't mean anything, but GitHub askes for this.

Reply:
new lines removed